mscore/scoreview.cpp

```diff
@@ -3658,7 +3658,7 @@ void ScoreView::addSlur(ChordRest* cr1, ChordRest* cr2, const Slur* slurTemplate
             _score->inputState().setSlur(slur);
             ss->setSelected(true);
             }
-      else if (switchToSlur) {
+      else if (switchToSlur && score()->selection().isSingle()) {
```

Comment:
This change seems to not apply to current master anymore, or to end up in a not yet implemented part of the code:

    if (m_noteInput->isNoteInputMode()) {
        m_noteInput->addSlur(slur);
    } else if (!secondChordRest) {
        NOT_IMPLEMENTED;
        //startEditMode(ss);
    }

No idea how to deal with this.